homeassistant/core.py

```diff
@@ -1662,11 +1663,10 @@ class Service:
 
     def __init__(
         self,
-        func: Callable[[ServiceCall], Coroutine[Any, Any, None] | None],
+        func: Callable[[ServiceCall], Coroutine[Any, Any, ServiceResult] | None],
         schema: vol.Schema | None,
         domain: str,
         service: str,
-        context: Context | None = None,
```

Comment:
Why is context removed?

Reply:
My impression is this is dead code.  In a previous revision (reverted) I was adding more things to this function signature and was doing this cleanup at the same time.. I can further revert this or pull into a separate PR if that is preferred.